nuxeo-compound-documents-web/elements/nuxeo-compoud-documents.js

```diff
@@ -0,0 +1,37 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+
+/**
+ * `nuxeo-compound-documents` is just a placeholder for elements.
+ * @memberof Nuxeo
+ */
+class NuxeoCompoundDocuments {
+  static get template() {
+    return html``;
+  }
+
+  static get is() {
+    return 'nuxeo-compound-documents';
+  }
+
+  static get properties() {
+    return {};
+  }
+}
+
+customElements.define(NuxeoCompoundDocuments.is, NuxeoCompoundDocuments);
```

Comment:
Not sure. I added this element because if we don't have any element to lint, `polymer lint` is trying to lint other files that are not even defined in the sources property of `polymer.json` 🤔

Reply:
I think we can remove this and disable polymer lint. We can maybe add it back once we contribute the new document tree. WDYT?